hs_core/management/commands/check_s3_files.py

```diff
@@ -17,11 +17,11 @@
 
 from hs_composite_resource.models import CompositeResource
 from hs_core.models import BaseResource
-from hs_core.management.utils import check_irods_files, check_for_dangling_irods
+from hs_core.management.utils import check_s3_files, check_for_dangling_s3
 
 
 class Command(BaseCommand):
-    help = "Check synchronization between iRODS and Django."
+    help = "Check synchronization between S3 and Django."
```

Comment:
can we test this management command and see if it actually still runs to completion?